src/components/CreateCoupons/index.jsx

```diff
@@ -0,0 +1,226 @@
+import React, { useState } from "react";
+import "./style.css";
+import { Typography, Button, Row, Col, Avatar, message } from "antd";
+import { InfoCircleOutlined } from "@ant-design/icons";
+import { useTranslation } from "react-i18next";
+import firebase from "../../firebaseConfig";
+
+const db = firebase.firestore();
+const timestamp = firebase.firestore.FieldValue.serverTimestamp;
+const { Title, Text } = Typography;
+
+const CreateCoupons = ({ isTesting }) => {
+  const { t } = useTranslation();
+  const [count, setCount] = useState(1);
+  const [newCoupons, setNewCoupons] = useState({
+    compName: "",
+    amounts: {
+      25: false,
+      50: false,
+      75: false,
+      100: false,
+      150: false,
+      200: false,
+    },
+  });
+  const [ghosted, setGhosted] = useState({
```

Comment:
can we add comments to these states, it was hard for me to understand what does it do right away